examples/Bremsstrahlung/include/simulation_defines/param/componentsConfig.param

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2013-2017 Axel Huebl, Anton Helm, Richard Pausch
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+#pragma once
+
+
+namespace picongpu
```

Comment:
I asked this in the [last review](https://github.com/ComputationalRadiationPhysics/picongpu/pull/1818#discussion_r100840162).
Are there any changes in this files, compared to the default params?